purchase_order_line_schedule/views/purchase_order_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="date_approve" position="after">
+                <field name="has_qty_to_schedule" invisible="True" />
+            </field>
+            <xpath
+                expr="//notebook//field[@name='order_line']/tree/field[@name='date_planned']"
+                position="after"
+            >
+                <field name="schedule_line_ids" invisible="1" />
+                <button
+                    string="Delivery Schedule"
+                    name="action_open_schedule_lines"
+                    type="object"
+                    icon="fa-list"
+                    optional="hide"
```

Comment:
Why hiding this massive important feature by default?

Reply:
@CasVissers-360ERP because the field 'date_planned' is also optional. Either we show or hide both to keep consistency.

Reply:
Show both makes the module more intuitive in my opinion.

Reply:
I am adding only the button. The other field is anyway useless when you use this module.